src/Lykke.Job.RabbitMqMonitoring/PeriodicalHandlers/CheckRabbitMqHandler.cs

```diff
@@ -34,29 +35,60 @@ public class CheckRabbitMqHandler : TimerPeriod
         {
             var tasks = new List<Task>();
 
-            foreach (var rabbitMq in _rabbitMqConnections)
+            foreach (var connectionSettings in _rabbitMqConnectionSettings)
             {
-                tasks.Add(ProcessConnection(rabbitMq));
+                tasks.Add(ProcessConnectionAsync(connectionSettings));
             }
 
             await Task.WhenAll(tasks);
         }
 
-        private async Task ProcessConnection(RabbitMqConnection rabbitMq)
+        private async Task ProcessConnectionAsync(RabbitMqConnectionSettings connectionSettings)
         {
             try
             {
-                var queues = await _rabbitMqManagementService.GetQueuesAsync(rabbitMq.Url, rabbitMq.Username, rabbitMq.Password);
+                var queues = await _rabbitMqManagementService.GetQueuesAsync(connectionSettings.Url, connectionSettings.Username, connectionSettings.Password);
 
-                foreach (var queue in queues.Where(item => item.Messages >= _maxMessagesCount))
+                foreach (var queue in queues)
                 {
-                    await _log.WriteMonitorAsync(new Uri(rabbitMq.Url).Host, string.Empty, string.Empty, $"Queue '{queue.Name}' contains {queue.Messages} messages");
+                    await ProcessQueueAsync(connectionSettings, queue);
                 }
             }
             catch (Exception ex)
             {
-                await _log.WriteErrorAsync(nameof(Execute), rabbitMq.Url, ex);
+                await _log.WriteErrorAsync(nameof(Execute), connectionSettings.Url, ex);
             }
         }
+
+        private async Task ProcessQueueAsync(RabbitMqConnectionSettings connectionSettings, RabbitMqQueue queue)
+        {
+            var queueSettings = TryGetQueueSettings(connectionSettings, queue);
+            var maxMessagesCount = queueSettings?.MaxMessagesCount ??
+                                   connectionSettings.MaxMessagesCount ?? 
+                                   _maxMessagesCount;
+
+            if (queue.Messages >= maxMessagesCount)
+            {
+                var title = connectionSettings.Title ?? new Uri(connectionSettings.Url).Host;
+                
+                await _log.WriteMonitorAsync(
+                    title,
+                    string.Empty,
+                    string.Empty,
+                    $"Queue '{queue.Name}' contains {queue.Messages} messages");
```

Comment:
Didn't you forget .count()?

Reply:
Nope. `Messages` is int. Name is defined by the RabbitMq management API contract